dashboard/about.md

```diff
@@ -64,27 +64,19 @@ Data for the dashboard is pulled from these sources on Mondays and Tuesdays.
 *   **Epidemiological week (MMWR week)**
     
     Week that starts on a Sunday. If the day on which the forecast is being made is a Sunday or Monday, the next epidemiological week is the week that starts on that Sunday (going back a day if it is Monday). If the forecast is being made on Tuesday-Saturday, the next epidemiological week is the week that starts on the subsequent Sunday, following [CDC convention](https://wwwn.cdc.gov/nndss/document/MMWR_week_overview.pdf).
-    
-*   **Point Forecast**
```

Comment:
Why remove this?

Reply:
I felt this description was trying to do two things. 1.) Define what a point estimate is in general and 2.) Explain how we decide what the point estimate is for submitted forecasts.

As a definition, it's a common statistical term. If we're not going to define "quantile", "WIS" or "coverage" which are all more complicated, then it seemed out of place. Also, I felt the definition is wrong, it's not necessarily the most likely value.

In terms of our interpretation, I thought that was valuable, but I moved it to the "Notes on the Data" section.

Reply:
Ok makes sense!